intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/element/version_1_1/YangYangCharCheck_v1_1.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *   Copyright (c) 2021-2022 PANTHEON.tech, s.r.o. All rights reserved.
+ *
+ *   This program and the accompanying materials are made available under the
+ *   terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ *   and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ */
+
+package tech.pantheon.yanginator.plugin.annotator.element.version_1_1;
+
+import com.intellij.lang.annotation.AnnotationHolder;
+import com.intellij.lang.annotation.HighlightSeverity;
+import com.intellij.psi.PsiElement;
+import org.jetbrains.annotations.NotNull;
+import tech.pantheon.yanginator.plugin.annotator.element.AbstractYangStmtCheck;
+import tech.pantheon.yanginator.plugin.psi.YangVchar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class YangYangCharCheck_v1_1 extends AbstractYangStmtCheck {
+    @Override
+    public boolean isApplicable(@NotNull final PsiElement element) {
+        return element instanceof YangVchar;
+    }
+
+    @Override
+    public void performCheck(@NotNull final PsiElement element, @NotNull final AnnotationHolder holder) {
+        final List<String> stringList = new ArrayList<>(List.of("%xD800-DFFF",
+                "%xFDD0-FDEF", "%xFFFE-FFFF", "%x1FFFE-1FFFF",
+                "%x2FFFE-2FFFF", "%x3FFFE-3FFFF", "%x4FFFE-4FFFF",
+                "%x5FFFE-5FFFF", "%x6FFFE-6FFFF", "%x7FFFE-7FFFF",
+                "%x8FFFE-8FFFF", "%x9FFFE-9FFFF", "%xAFFFE-AFFFF",
+                "%xBFFFE-BFFFF", "%xCFFFE-CFFFF", "%xDFFFE-DFFFF",
+                "%xEFFFE-EFFFF", "%xFFFFE-FFFFF", "%x10FFFE-10FFFF"));
+        final List<Character> charList = stringsToChars(stringList);
```

Comment:
can we make this static, so it is not created each time perform check is called

Reply:
Fixed